src/js/utils/PodUtil.js

```diff
@@ -0,0 +1,42 @@
+var PodUtil = {
+
+  /**
+   * Returns true if the container given matches
+   * the free-text given.
+   *
+   * @param {PodContainer} container - The Container to test
+   * @param {string} text - The text to test against
+   * @returns {boolean} - Returns true if the test passes
+   */
+  isContainerMatchingText(container, text) {
+    if (!text) {
+      return true;
+    }
+
+    return container.getName().indexOf(text) >= 0;
```

Comment:
@orlandohohmeier he refers to hist modification here : https://github.com/dcos/dcos-ui/pull/1068/commits/ce12e9b10635e4c92415ed90dcf750505110b104 (that breaks this PR).

I will re-introduce `getName()` and it will call `getId()` for now. However I wanted this separation because `getId()` should be used on server communication, while `getName()` on the UI.